src/board_repository.py

```diff
@@ -0,0 +1,37 @@
+import os
+from to_do_item import ToDoItem
+from pymongo import MongoClient
+from bson import ObjectId
+from datetime import datetime
+
+class BoardRepository:
+    def __init__(self):
+        mongo_connection_string = os.environ['MONGO_CONNECTION_STRING']
+        mongo_db_name = os.environ['MONGO_DB_NAME']
+
+        self.mongoClient = MongoClient(mongo_connection_string)
+        self.db = self.mongoClient[mongo_db_name]
+        self.itemsCollection = self.db.items
+
+    def get_items(self):
+        items = self.itemsCollection.find()
+        return [ToDoItem(str(item['_id']), item['name'], item['status'], item['lastModified']) for item in items]
```

Comment:
Really minor, but this might be a good candidate for a class/static method on the ToDoItem class, like you had for `from_trello_data`